src/backend/aspen/api/views/phylo_runs.py

```diff
@@ -155,14 +160,18 @@ async def kick_off_phylo_run(
 
 
 async def get_serializable_runs(
-    db: AsyncSession, az: AuthZSession, privilege="read", run_id=None
+    db: AsyncSession, az: AuthZSession, pathogen, privilege="read", run_id=None
 ):
     query = await az.authorized_query(privilege, PhyloRun)
     query = query.options(
         joinedload(PhyloRun.outputs.of_type(PhyloTree)),  # type: ignore
         joinedload(PhyloRun.user),  # For Pydantic serialization
         joinedload(PhyloRun.group),  # For Pydantic serialization
     )
+    # TODO - DECOVIDIFY - remove the None check!
+    query = query.filter(
+        or_(PhyloRun.pathogen == pathogen, PhyloRun.pathogen_id == None)  # noqa: E711
+    )
```

Comment:
Again with the lame `or` clause, this will need to get updated once our migration is complete.